types/react-native/index.d.ts

```diff
@@ -5,6 +5,7 @@
 //                 Kyle Roach <https://github.com/iRoachie>
 //                 Tim Wang <https://github.com/timwangdev>
 //                 Kamal Mahyuddin <https://github.com/kamal>
+//                 Nikolas LeBlanc <https://github.com/nikolasleblanc>
```

Comment:
Just wanting to make sure that you understand that by adding this line means you’ll be signing up as a maintainer and will get notifications for all issues/PRs.